nosepipe.py

```diff
@@ -15,7 +15,7 @@
 
 import nose.plugins
 
-__version__ = "0.5"
+__version__ = "0.5-post3"
```

Comment:
You can leave out any version changes, as those will be done when a new release is made - right now this might conflict as 0.6 has been released.